tests/integration/test_requirements.py

```diff
@@ -101,11 +105,12 @@ def test_requirements_generates_requirements_from_lockfile_from_a_category(pipen
         assert f'{test_packages[0]}=={test_packages[1]}' not in c.stdout
         assert f'{dev_packages[0]}=={dev_packages[1]}' in c.stdout
 
-        d = p.pipenv('requirements --category test')
+        d = p.pipenv('requirements --category test,doc')
```

Comment:
With the other click CLI categories, this would be: `--categories"test, doc"`

Reply:
Updated.

Reply:
Updated the PR with the tests.